calibrate.py

```diff
@@ -135,103 +128,128 @@ class Main:
 
     def __init__(self):
         self.args = vars(parse_args())
```

Comment:
```suggestion
        self.args = parse_args()
```
Removing `vars` will allow to access parameters like `self.focus_value = self.args.rgbLensPosition` instead of `self.focus_value = self.args['rgbLensPosition']`

Reply:
Oohh. That's why I was not able to do that one. Thanks